server/grpc_service.go

```diff
@@ -1685,3 +1686,84 @@ func checkStream(streamCtx context.Context, cancel context.CancelFunc, done chan
 	}
 	<-done
 }
+
+const globalConfigPath = "/global/config/"
+
+func (s *Server) StoreGlobalConfig(ctx context.Context, request *pdpb.StoreGlobalConfigRequest) (*pdpb.StoreGlobalConfigResponse, error) {
```

Comment:
You need to update your branch with the latest master.

```suggestion
func (s *GrpcServer) StoreGlobalConfig(ctx context.Context, request *pdpb.StoreGlobalConfigRequest) (*pdpb.StoreGlobalConfigResponse, error) {
```

Reply:
ok i will do that